mangaki/mangaki/choices.py

```diff
@@ -56,3 +56,15 @@
     ('same_setting', 'Univers commun'),
     ('other', 'Spécial')
 )
+
+SUGGESTION_PROBLEM_CHOICES = (
+    ('title', 'Le titre n\'est pas le bon'),
```

Comment:
It's alright to use `"`-quotes to avoid escaping, you don't have to make this change but next time, you don't have to suffer :D.